google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -407,6 +416,40 @@ def storage_class= new_storage_class
           patch_gapi! :storage_class
         end
 
+        ##
+        # Whether Autoclass is enabled for the bucket.
+        #
+        # @return [Boolean]
+        #
+        def autoclass_enabled
+          @gapi.autoclass&.enabled
+        end
+
+        ##
+        # Toggle time of the autoclass
+        #
+        # @return [DateTime]
+        #
+        def autoclass_toggle_time
+          @gapi.autoclass&.toggle_time
+        end
+
+        ##
+        # Updates bucket's autoclass configuration. This defines the default class for objects in the
+        # bucket and down/up-grades the storage class of objects based on the access patterns.
+        # Accepted values are `:false`, and `:true`.
+        # rubocop:todo update the link
+        # For more information, see [Storage
+        # Classes](https://cloud.google.com/storage/docs/add_the_endpoint).
```

Comment:
This is the docs link that will be used. Please feel free to update here
​​https://cloud.google.com/storage/docs/using-autoclass

Reply:
Merged both the PRs and got the link.